include/internal/propertyerr.h

```diff
@@ -8,10 +8,10 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_PROPERR_H
-# define HEADER_PROPERR_H
+#ifndef OPENSSL_PROPERR_H
+# define OPENSSL_PROPERR_H
```

Comment:
hmm, why is this not OPENSSL_PROPERTYERR_H or OPENSSL_INTERNAL_PROPERTYERR_H ?

Reply:
changed to OPENSSL_INTERNAL_PROPERTYERR_H